docs/manual/working/scalaGuide/main/tests/ScalaFunctionalTestingWithScalaTest.md

```diff
@@ -119,7 +125,7 @@ If you are using an SQL database, you can replace the database connection with a
 
 ## Testing WS calls
 
-If you are calling a web service, you can use [`WSTestClient`](api/scala/play/api/test/WsTestClient.html).  There are two calls available, `wsCall` and `wsUrl` that will take a Call or a string, respectively.  Note that they expect to be called in the context of `WithApplication`.
+If you are calling a web service, you can use [`WSTestClient`](api/scala/play/api/test/WsTestClient.html).  There are two calls available, `wsCall` and `wsUrl` that will take a Call or a string, respectively.  Note that they expect to be called in the context of a running application.
```

Comment:
This should probably be deprecated since it uses global state. Do we have an alternative that doesn't require global state?